packages/psd/src/sections/LayerAndMaskInformation/AdditionalLayerInfo/index.ts

```diff
@@ -39,6 +52,7 @@ export function readAdditionalLayerInfo(
   // Position the cursor at the end of the ALI block
   const remainingBytes = size - (cursor.position - prevPosition);
   cursor.pass(remainingBytes);
+  cursor.padding(size, 4);
```

Comment:
@scoiatael This is a bit late, but are you sure this is correct?

I used to have the same code before I discovered one of my sample PSD files that failed here. I had to remove this line in f4fc9497d2826aeaed17910194dc9e8f7130ebe9. (You might want to read the commit message)

Now that we merged this PR, it seems that the bug has regressed. Unfortunately, I could not provide a test to cover this case because the affected PSD file is from a private source and cannot be published as open source; and I could not produce another PSD file myself.

Your PR added [code that parses the Global Additional Layer Information](https://github.com/opendesigndev/psd-ts/blob/6135422657c8049a8dda07269b6d28c99fa0d042/packages/psd/src/sections/LayerAndMaskInformation/readLayerRecordsAndChannels.ts#L206-L208) (what we internally call "tagged blocks"). Perhaps we could move the padding code there?